components/d2l-organization-admin-list/d2l-organization-admin-list.js

```diff
@@ -191,6 +196,25 @@ class AdminList extends EntityMixinLit(LocalizeMixin(LitElement)) {
 						margin-bottom: 24px;
 						width: 100%;
 					}
+
+				}
+
+				@media (max-width: 349px) {
+					.learningpath-title{
+						width: 100px;
+					}
+
+				}
+
+				@media (min-width: 350px) and  (max-width: 474px){
+					.learningpath-title{
+						width: 150px;
```

Comment:
```suggestion
						max-width: 150px;
```

Sorry, I wasn't clear enough here. We should be using `max-width` for the actual learning path title.

Reply:
will test and fix